lint.ts

```diff
@@ -13,9 +13,39 @@ let badExit = false;
 // error reports an error to the console and sets badExit to true
 // so that the process will exit with a non-zero exit code.
 const error = (...data: any[]) => {
-    console.error(...data);
-    badExit = true;
-}
+  console.error(...data);
+  badExit = true;
+};
+
+const verifyCodeBlocks = (
+  tokens: marked.Token[],
+  res = {
+    codeIsTF: false,
+    codeIsHCL: false,
+  }
+) => {
+  for (const token of tokens) {
+    // Check in-depth.
+    if (token.type === "list") {
```

Comment:
Review: There may be other in-depth cases that are unhandled, but this was present in `slackme` README, at least.